homeassistant/components/switcher_kis/sensor.py

```diff
@@ -98,30 +98,31 @@ class SwitcherSensorEntity(CoordinatorEntity, SensorEntity):
 
     def __init__(
         self,
-        wrapper: SwitcherDeviceWrapper,
+        coordinator: SwitcherDeviceWrapper,
```

Comment:
This is confusing, since we often use `coordinator` to refer to `DataUpdateCoordinator`. Is there a reason to not use a name more consistent with its type?

Reply:
The `SwitcherDeviceWrapper` is based on `DataUpdateCoordinator` but it is a wrapper on it with additional functionality , this is why I called it `wrapper` (I use this name for multiple integrations I am maintaining).

@MartinHjelmare reviewed on https://github.com/home-assistant/core/pull/52316 and asked to rename it to `coordinator` since it will be self assigned by the `CoordinatorEntity` at init.

Personally I prefer to leave the name as `wrapper` since that what it is.

Reply:
Hmm, okay. I'll defer to @MartinHjelmare.

Reply:
It's still a coordinator since we extend that base class. I would have named the class `SwitcherCoordinator` or similar. When inheriting a base class you always add some features. Otherwise there's no need to make a child class.

Reply:
Renamed `SwitcherDeviceWrapper` to `SwitcherDataUpdateCoordinator` and changed all places with `wrapper` to `coordinator`